### PR TITLE
task solution

### PR DESCRIPTION
**Pull request by vladyslav-harkusha**
[DEMO LINK](https://vladyslav-harkusha.github.io/react_static-list-of-todos/)

### CODE REVIEW

**Review by imaksweb — APPROVED** [review]
Nice work 🔥

File: src/components/UserInfo/UserInfo.tsx [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
```
Comment (Denchepornyuk) [inline src/components/UserInfo/UserInfo.tsx:12]: it's better to not render `a` if user is `null`

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]

File: src/components/TodoList/TodoList.tsx [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo todo={todo} key={todo.id} />
+    ))}
+  </section>
```
Comment (OleksiiBlezniuk) [inline src/components/TodoList/TodoList.tsx:15]: Use semantic tags for list and its items

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]

File: src/components/TodoList/TodoList.tsx (hunk 2 above)
Comment (vladyslav-harkusha) [reply src/components/TodoList/TodoList.tsx]: But in the task, we only have to split the given markup into components, and don't change class names

Comment (OleksiiBlezniuk) [reply src/components/TodoList/TodoList.tsx]: Okay, probably it's not so strong option, but still using of semantic tags is better then not using of them, tags is not a classes, using of `ul` and `li` tags wouldn't break your layout or tests (in this case)

**Review by OleksiiBlezniuk — APPROVED** [review]

**Pull request closed without merging**